### ISSUE TITLE
Keyboard shortcut to switch between channels

### ISSUE THREAD

**Issue by geekgonecrazy** [RocketChat/Rocket.Chat#4256]
We need a shortcut to switch between channels.  Something like `alt+up` to go up in list `alt+down` to go down list.

**Comment by masonfox** [comment]
Or `Control + #` depending on what channels you have

**Comment by emgee** [comment]
See also https://github.com/RocketChat/Rocket.Chat/issues/4255.

**Comment by geekgonecrazy** [comment]
@masonfox only problem we might have with `ctrl + #` is our desktop app uses this to switch between servers.  :(  so if we did that one... we'd have to switch the desktop app.

**Comment by davistan** [comment]
Hope you guys don't mind me adding to here as I believe it is related.

Some shortcut keys on show/hide the right panel with Escape key as the global hide key will be really helpful. Such as the info, search, users panel, etc.

**Comment by tsdgeos** [comment]
Another vote for Alt+up and Alt+down

**Comment by kaicode** [comment]
👍 Another vote here for this feature.
Something used as often as a messaging application should be capable of keyboard navigation!
Thanks for such a great product 🥇

**Comment by jhermann** [comment]
See also https://github.com/RocketChat/Rocket.Chat.Electron/issues/221

**Comment by TwizzyDizzy** [comment]
@rocket-cat close

I'm closing this since I think this has been implemented with the introduction of `[CTRL] + [K]`. If anyone disagrees, please get back to me! :)

Cheers
Thomas

**Comment by tsdgeos** [comment]
I disagree, Ctrl+K is very cumbersome, you have to actually type the name of the channel, instead of just move thorugh the list, also Ctrl+k changes the channel list so it's visually jarring

**Comment by TwizzyDizzy** [comment]
@rocket-cat open

@tsdgeos agreed. Re-open.

**Comment by cmrd-senya** [comment]
I created a PR  #10366 for this feature, but I need some feedback from more experienced contributors in order to complete it.

**Comment by piotrkochan** [comment]
For the web alt + up/down is ok, but for Electron App it would be better to bind ctrl + tab and ctrl + shift + tab.
